testing/go/e2e_test.go

```diff
@@ -10,16 +10,54 @@ import (
 func TestEndToEnd(t *testing.T) {
 	log.Println("Starting E2E test...")
 
-	log.Println("kubectl get crd")
-	c, b := exec.Command("kubectl", "get", "crd"), new(strings.Builder)
+	// Check that the BTPOperator resource is created and has the expected status.
+	log.Println("kubectl get BTPOperator")
+	c, b := exec.Command("kubectl", "get", "BTPOperator"), new(strings.Builder)
 
 	c.Stdout = b
 	c.Run()
 	out := b.String()
 
 	log.Println(out)
 
-	if len(out) == 0 {
-		t.Error("No CRD found")
+	if !strings.Contains(out, "Running") {
```

Comment:
Where do you expect to get "Running"? we do not have such string, see: https://github.com/kyma-project/btp-manager/blob/main/docs/operations.md#conditions